ta/crypt/cryp_taf.c

```diff
@@ -47,104 +49,121 @@ TEE_Result ta_entry_allocate_operation(uint32_t param_type, TEE_Param params[4])
 	res = TEE_AllocateOperation(&op,
 				    params[0].value.b, params[1].value.a,
 				    params[1].value.b);
-	params[0].value.a = (uint32_t) op;
+	params[0].value.a = (uintptr_t)op;
```

Comment:
A pointer passed as handles to TAs are converted with `tee_svc_kaddr_to_uref()` to make it a low number.